java/ql/src/semmle/code/java/dataflow/internal/ContainerFlow.qll

```diff
@@ -88,6 +91,290 @@ class ContainerType extends RefType {
+        "java.util;Collection;true;parallelStream;();;Element of Argument[-1];Element of ReturnValue;value",
+        "java.util;Collection;true;stream;();;Element of Argument[-1];Element of ReturnValue;value",
+        "java.util;Collection;true;toArray;;;Element of Argument[-1];ArrayElement of ReturnValue;value",
+        "java.util;Collection;true;add;;;Argument[0];Element of Argument[-1];value",
+        "java.util;Collection;true;addAll;;;Element of Argument[0];Element of Argument[-1];value",
+        "java.util;List;true;get;(int);;Element of Argument[-1];ReturnValue;value",
+        "java.util;List;true;listIterator;;;Element of Argument[-1];Element of ReturnValue;value",
+        "java.util;List;true;remove;(int);;Element of Argument[-1];ReturnValue;value",
+        "java.util;List;true;set;;;Element of Argument[-1];ReturnValue;value",
+        "java.util;List;true;set;;;Argument[1];Element of Argument[-1];value",
+        "java.util;List;true;subList;;;Element of Argument[-1];Element of ReturnValue;value",
+        "java.util;List;true;add;(int,Object);;Argument[1];Element of Argument[-1];value",
+        "java.util;List;true;set;(int,Object);;Argument[1];Element of Argument[-1];value",
+        "java.util;List;true;addAll;(int,Collection);;Element of Argument[1];Element of Argument[-1];value",
+        "java.util;Vector;true;elementAt;(int);;Element of Argument[-1];ReturnValue;value",
+        "java.util;Vector;true;elements;();;Element of Argument[-1];Element of ReturnValue;value",
+        "java.util;Vector;true;firstElement;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Vector;true;lastElement;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Vector;true;addElement;(Object);;Argument[0];Element of Argument[-1];value",
+        "java.util;Vector;true;insertElementAt;(Object,int);;Argument[0];Element of Argument[-1];value",
+        "java.util;Vector;true;setElementAt;(Object,int);;Argument[0];Element of Argument[-1];value",
+        "java.util;Vector;true;copyInto;(Object[]);;Element of Argument[-1];ArrayElement of Argument[0];value",
+        "java.util;Stack;true;peek;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Stack;true;pop;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Stack;true;push;(Object);;Argument[0];Element of Argument[-1];value",
+        "java.util;Queue;true;element;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Queue;true;peek;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Queue;true;poll;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Queue;true;remove;();;Element of Argument[-1];ReturnValue;value",
+        "java.util;Queue;true;offer;(Object);;Argument[0];Element of Argument[-1];value",
+        "java.util;Deque;true;descendingIterator;();;Element of Argument[-1];ReturnValue;value",
```

Comment:
Other iterators seem to use `Element of ...`

Reply:
yeah, that's a typo...